### PR TITLE
fix(serverless-hub) Refactor logging and fix tests

### PR DESCRIPTION
**Pull request by chrismaree**
**Motivation**

After the most recent PRs coming in there were was a small issue introduced from a bad merge in the logger in the hub. This PR addresses this.

Additionally, the hub & spoke unit tests has a few small issues in them as they are not checked into CI. This is not ideal and will be addressed in issue #2001

### CODE REVIEW

File: packages/financial-templates-lib/src/logger/SlackTransport.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ function slackFormatter(info) {
               type: "section",
               text: {
                 type: "mrkdwn",
-                text: `    - _${subKey}_: ${JSON.stringify(info[key][subKey], null, 2)}\n`
+                text: `    - _${subKey}_: ${JSON.stringify(info[key][subKey])}\n`
```
Comment (chrismaree) [inline packages/financial-templates-lib/src/logger/SlackTransport.js:96]: this change was made to clean up slack logs a bit to not wrap on multiple lines

File: packages/core/scripts/severless-cloud-run/CloudRunHub.js [hunk 2]
```diff
@@ -96,26 +96,26 @@ hub.post("/", async (req, res) => {
     });
 
     // Validate that the promises returned correctly. If ANY have error, then catch them and throw them all together.
-    let errorOutputs = [];
-    let validOutputs = [];
+    let errorOutputs = {};
+    let validOutputs = {};
     results.forEach((result, index) => {
-      if (result.status == "rejected" || result?.value?.execResponse?.error || result?.reason?.code != "200") {
+      if (result.status == "rejected" || result?.value?.execResponse?.error || result?.reason?.code == "500") {
```
Comment (chrismaree) [inline packages/core/scripts/severless-cloud-run/CloudRunHub.js:102]: I reverted this back to explicitly checking for code 500 as the spoke always errors with this code. checking `!="200"` would incorrectly detect some situations as errors as a status code is not always sent.

File: packages/core/scripts/severless-cloud-run/CloudRunHub.js [hunk 3]
```diff
@@ -96,26 +96,26 @@ hub.post("/", async (req, res) => {
...
         // If the child process in the spoke crashed it will return 500 (rejected). OR If the child process exited
         // correctly but contained an error.
-        errorOutputs.push({
+        errorOutputs[Object.keys(configObject)[index]] = {
           status: result.status,
```
Comment (chrismaree) [inline packages/core/scripts/severless-cloud-run/CloudRunHub.js:106]: this is the main fix in this PR.

File: packages/core/scripts/severless-cloud-run/test/CloudRunHub.js [hunk 4]
```diff
@@ -107,7 +107,8 @@ contract("CloudRunHub.js", function(accounts) {
       sponsorDisputeRewardPct: { rawValue: toWei("0.1") },
       disputerDisputeRewardPct: { rawValue: toWei("0.1") },
       minSponsorTokens: { rawValue: toWei("1") },
-      timerAddress: (await Timer.deployed()).address
+      timerAddress: (await Timer.deployed()).address,
+      excessTokenBeneficiary: "0x0000000000000000000000000000000000000000"
```
Comment (chrismaree) [inline packages/core/scripts/severless-cloud-run/test/CloudRunHub.js:111]: this was added in after the merge `excessTokenBeneficiary`

File: packages/core/scripts/severless-cloud-run/test/CloudRunHub.js [hunk 5]
```diff
@@ -164,7 +165,7 @@ contract("CloudRunHub.js", function(accounts) {
 
     const hubConfig = {
       testCloudRunMonitor: {
-        cloudRunCommand: "yarn monitors --network test",
+        cloudRunCommand: "yarn --silent monitors --network test",
```
Comment (chrismaree) [inline packages/core/scripts/severless-cloud-run/test/CloudRunHub.js:168]: cleaner log outputs. doesent really affect the tests but is more consistent

File: packages/core/scripts/severless-cloud-run/test/CloudRunHub.js [hunk 6]
```diff
@@ -310,31 +311,38 @@ contract("CloudRunHub.js", function(accounts) {
...
+    assert.isTrue(lastSpyLogIncludes(hubSpy, "Some spoke calls returned errors")); // The hub should have exited correctly.
     assert.equal(lastSpyLogLevel(hubSpy), "error"); // most recent log level should be "error"
-    assert.equal(responseObject.output.errorOutputs.length, 2); // should be 2 errors
-    assert.equal(responseObject.output.validOutputs.length, 1); // should be 1 valid output
+    assert.equal(Object.keys(responseObject.output.errorOutputs).length, 2); // should be 2 errors
+    assert.equal(Object.keys(responseObject.output.validOutputs).length, 1); // should be 1 valid output
 
     // Check the valid outputs.
-    assert.equal(responseObject.output.validOutputs[0].botIdentifier, "testServerlessMonitor"); // Check that the valid output is the expected bot
+    assert.equal(responseObject.output.validOutputs["testServerlessMonitor"].botIdentifier, "testServerlessMonitor"); // Check that the valid output is the expected bot
```
Comment (chrismaree) [inline packages/core/scripts/severless-cloud-run/test/CloudRunHub.js:323]: changes made as the errors are now key-value paired for easy consumption

File: packages/core/scripts/severless-cloud-run/CloudRunHub.js (hunk 2 above)
Comment (nicholaspai) [reply packages/core/scripts/severless-cloud-run/CloudRunHub.js:102]: You might want to switch this to only checking if the code starts with "2"?

**Review by nicholaspai — APPROVED** [review]
Your comments helpedme review faster, thanks looks good!

**Pull request merged**